src/Databases/IDatabaseReplicating.h

```diff
@@ -0,0 +1,13 @@
+#pragma once
+
+namespace DB
+{
+
+class IDatabaseReplicating
+{
+public:
+    virtual void stopReplication() = 0;
+    virtual ~IDatabaseReplicating() = default;
+};
```

Comment:
I'm not sure that we need separate class for this, maybe just add methods like `hasReplicatingThread` (that returns `false` by default) and `stopReplication()` (that throws `NOT_IMPLEMENTED`/`LOGICAL_ERROR`) to `IDatabase`?

Reply:
Fixed.